[2023-08-31T02:26:01.864+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_with_xcom/run_id=scheduled__2023-08-30T00:10:00+00:00/task_id=bash_pull permission to 509
[2023-08-31T02:26:01.865+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_with_xcom/run_id=scheduled__2023-08-30T00:10:00+00:00/task_id=bash_pull permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_with_xcom/run_id=scheduled__2023-08-30T00:10:00+00:00/task_id=bash_pull'
[2023-08-31T02:26:01.908+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_with_xcom/run_id=scheduled__2023-08-30T00:10:00+00:00/task_id=bash_pull permission to 509
[2023-08-31T02:26:01.911+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_with_xcom/run_id=scheduled__2023-08-30T00:10:00+00:00/task_id=bash_pull permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_with_xcom/run_id=scheduled__2023-08-30T00:10:00+00:00/task_id=bash_pull'
[2023-08-31T02:26:02.059+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dags_bash_with_xcom.bash_pull scheduled__2023-08-30T00:10:00+00:00 [queued]>
[2023-08-31T02:26:02.068+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dags_bash_with_xcom.bash_pull scheduled__2023-08-30T00:10:00+00:00 [queued]>
[2023-08-31T02:26:02.069+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-31T02:26:02.103+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): bash_pull> on 2023-08-30 00:10:00+00:00
[2023-08-31T02:26:02.107+0000] {standard_task_runner.py:57} INFO - Started process 73 to run task
[2023-08-31T02:26:02.110+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dags_bash_with_xcom', 'bash_pull', 'scheduled__2023-08-30T00:10:00+00:00', '--job-id', '205', '--raw', '--subdir', 'DAGS_FOLDER/dags_bash_with_xcom.py', '--cfg-path', '/tmp/tmpzyst6q59']
[2023-08-31T02:26:02.113+0000] {standard_task_runner.py:85} INFO - Job 205: Subtask bash_pull
[2023-08-31T02:26:02.172+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=dags_bash_with_xcom/run_id=scheduled__2023-08-30T00:10:00+00:00/task_id=bash_pull permission to 509
[2023-08-31T02:26:02.173+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=dags_bash_with_xcom/run_id=scheduled__2023-08-30T00:10:00+00:00/task_id=bash_pull permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=dags_bash_with_xcom/run_id=scheduled__2023-08-30T00:10:00+00:00/task_id=bash_pull'
[2023-08-31T02:26:02.177+0000] {task_command.py:410} INFO - Running <TaskInstance: dags_bash_with_xcom.bash_pull scheduled__2023-08-30T00:10:00+00:00 [running]> on host 07fcf706bef2
[2023-08-31T02:26:02.337+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dags_bash_with_xcom' AIRFLOW_CTX_TASK_ID='bash_pull' AIRFLOW_CTX_EXECUTION_DATE='2023-08-30T00:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-30T00:10:00+00:00'
[2023-08-31T02:26:02.339+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-31T02:26:02.340+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo $PUSHED_VALUE && echo $RETURN_VALUE']
[2023-08-31T02:26:02.350+0000] {subprocess.py:86} INFO - Output:
[2023-08-31T02:26:02.351+0000] {subprocess.py:93} INFO - first_bash_message
[2023-08-31T02:26:02.352+0000] {subprocess.py:93} INFO - COMPLETE
[2023-08-31T02:26:02.352+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-31T02:26:02.364+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dags_bash_with_xcom, task_id=bash_pull, execution_date=20230830T001000, start_date=20230831T022602, end_date=20230831T022602
[2023-08-31T02:26:02.403+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-31T02:26:02.419+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
